.circleci/config.yml

```diff
@@ -140,16 +141,19 @@ jobs:
     parameters:
       py_version:
         type: string
-    executor: win/default
+    executor:
+      name: win/default
+      shell: bash.exe
     steps:
       - win:
           py_version: << parameters.py_version >>
       - run:
           name: Testing Hydra
           no_output_timeout: 10m
           command: |
-            $env:NOX_PYTHON_VERSIONS=<< parameters.py_version >>
-            $env:PYTHONIOENCODING="utf_8"
+            export NOX_PYTHON_VERSIONS="<< parameters.py_version >>"
+            export PYTHONIOENCODING="utf-8"
+            eval "$(conda shell.bash hook)"
```

Comment:
You have ` C:\tools\miniconda3\Scripts\conda.exe init powershell` here, which is why it does not work out of the box.
should probably change that to init for bash.

Reply:
Thx! I removed that.
`eval "$(conda shell.bash hook)"` this line seems to fix the init issue.

Reply:
you only need one of the two.
the init bash changes the shell to support conda permanently.
the eval you introduced changes the current session.

Reply:
I think `conda init bash` where we had `conda init powershell` is better.

Reply:
agreed. That was what i tried first. however the command failed with
`bash: line 1: C:toolsminiconda3Scriptsconda.exe: command not found`

failed run here https://app.circleci.com/pipelines/github/facebookresearch/hydra/4881/workflows/25dac692-1356-440d-af5c-ded57a2c62a9/jobs/35345
commit here https://github.com/facebookresearch/hydra/pull/1154/commits/f62120dcb36b9a21668c7bc9a1f10ea7469ab54d

found the workaround in this issue https://github.com/conda/conda-build/issues/3371

Reply:
`C:\tools\miniconda3\Scripts\conda.exe init powershell` has backslashes that are invalid in bash.
Try to switch to forward slashes.

Reply:
:O thx for the tip! will try it out.